roboteam_ai/src/skills/GoToPos.cpp

```diff
@@ -14,134 +14,51 @@ GoToPos::GoToPos(string name, bt::Blackboard::Ptr blackboard)
 
 /// Init the GoToPos skill
 void GoToPos::onInitialize() {
-    goToBall = properties->getBool("goToBall");
-    goBehindBall = properties->getBool("goBehindBall");
-
-    if (properties->hasVector2("distanceBehindBall")) {
-        distanceBehindBall = properties->getDouble("distanceBehindBall");
-    } else {
-        distanceBehindBall = 0.2;
+    if (properties->hasVector2("targetPos")) {
+        targetPos = properties->getVector2("targetPos");
     }
 
-    if (properties->hasVector2("Position")) {
-        targetPos = properties->getVector2("Position");
+    if (properties->hasDouble("maxVel")) {
```

Comment:
is position being replaced by maxvel here?

Reply:
No, position is being replaced by targetPos and maxVel is added